tests/expected/function-contract/mutable-references/return_mut_ref.expected

```diff
@@ -0,0 +1 @@
+VERIFICATION:- SUCCESSFUL
```

Comment:
I'd suggest also including a particular "assertion ..." line(s).

Reply:
I added a few lines, although the test is really elementary and doesn't have any interesting assertion.